vendor/modules.txt

```diff
@@ -558,16 +558,17 @@ go.opentelemetry.io/proto/otlp/collector/trace/v1
 go.opentelemetry.io/proto/otlp/common/v1
 go.opentelemetry.io/proto/otlp/resource/v1
 go.opentelemetry.io/proto/otlp/trace/v1
-# golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
-## explicit; go 1.11
+# golang.org/x/crypto v0.0.0-20211202192323-5770296d904e
+## explicit; go 1.17
```

Comment:
Looks like Go maintainers themselves no longer see this as "minimum version required", but just use it to force the go 1.17 format in go.mod 🙄 https://github.com/golang/crypto/commit/e9a32991a82ef02a1e74f495dcc0785239782bfe

```
Note that this does *not* prevent users with earlier go versions from
successfully building packages from this module.
```